internal/pkg/pluginmanager/manager.go

```diff
@@ -4,14 +4,19 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"os/exec"
 	"path/filepath"
 
 	"github.com/spf13/viper"
 
+	"github.com/merico-dev/stream/cmd/devstream/version"
 	"github.com/merico-dev/stream/internal/pkg/configloader"
+	md5helper "github.com/merico-dev/stream/internal/pkg/md5"
 	"github.com/merico-dev/stream/pkg/util/log"
 )
 
+const DTMRemoteMD5 = ".remote"
```

Comment:
The const name `DTMRemoteMD5` seems hard to understand, why is the remote md5 a string? Or conversely, what exactly does this string .remote mean?

Reply:
It is a dir name,  I renamed it to "DTMRemoteMD5Dir"